client/src/components/Button/Button.tsx

```diff
@@ -4,11 +4,15 @@ import React, { FunctionComponent } from 'react';
 import * as S from './Button.styles';
 
 interface ButtonProps {
+  type?: 'button' | 'submit' | 'reset';
   color?: string;
+  ariaLabel: string;
 }
 
-const Button: FunctionComponent<ButtonProps> = ({ color, children }) => (
-  <S.Button color={color}>{children}</S.Button>
+const Button: FunctionComponent<ButtonProps> = ({ type, color, ariaLabel, children }) => (
+  <S.Button type={type} color={color} aria-label={ariaLabel}>
```

Comment:
why button component doesn't contain any onClick handler?

Reply:
I guess none of them has any. For instance, in a form I've put handler onto the form itself and not the submit button. I also have another button called ButtonSwitch, and that one has a handler, because I use it in two places, as a theme switch and as a forecast/air-quality switch.

Reply:
i'd recommend to add onClick as far as it's a shared component. but anyway if you'll use recommended interface update, you can just write something like this:
<S.Button color={color} {...otherProps}>